CONTRIBUTING.md

```diff
@@ -59,17 +59,25 @@ By making a contribution to this project, I certify that:
 
 Then you just add a line to every git commit message:
 
-    Signed-off-by: Joe Smith <joe.smith@example.com>
+``` text
+Signed-off-by: Joe Smith <joe.smith@example.com>
+```
 
 Use your real name (sorry, no pseudonyms or anonymous contributions.)
 
+To ensure you are using the correct email address you can run the following command:
+
+``` bash
+git config user.email
```

Comment:
This command needs to be executed within a git repository directory, otherwise an error will be produced. 

Should we specify the `--global` option to that the config applies to all repositories? In addition, should we include an example email address? 

```shell
git config --global user.email joe.smith@example.com
```

Reply:
This has been fixed. Thanks for the catch.